client/src/components/PageLayout/Header.tsx

```diff
@@ -69,32 +68,42 @@ export const Header: React.FC = () => {
                 >
                   Menu
                 </MenuButton>
-                <MenuList paddingBlock={0}>
+                <MenuList paddingBlock={0} as="ul">
                   <Flex
                     flexDirection="column"
                     fontWeight="600"
                     borderRadius="5px"
                   >
-                    <NextLink passHref href="/chapters">
-                      <MenuItem as="a">Chapters</MenuItem>
-                    </NextLink>
+                    <MenuItem as="li" cursor="pointer">
+                      <Link width="100%" href="/chapters">
+                        Chapters
+                      </Link>
+                    </MenuItem>
 
-                    <NextLink passHref href="/events">
-                      <MenuItem as="a">Events</MenuItem>
-                    </NextLink>
+                    <MenuItem as="li" cursor="pointer">
+                      <Link width="100%" href="/events">
+                        Events
+                      </Link>
+                    </MenuItem>
 
                     {user && (
                       <Box borderBlock={'1px'} borderColor={'gray.85'}>
-                        <NextLink passHref href="/profile">
-                          <MenuItem as="a">Profile</MenuItem>
-                        </NextLink>
+                        <MenuItem as="li" cursor="pointer">
+                          <Link width="100%" href="/profile">
+                            Profile
+                          </Link>
+                        </MenuItem>
 
                         {checkPermission(user, Permission.ChaptersView) && (
-                          <NextLink passHref href="/dashboard/chapters">
-                            <MenuItem data-cy="menu-dashboard-link" as="a">
+                          <MenuItem
+                            data-cy="menu-dashboard-link"
+                            as="li"
+                            cursor="pointer"
+                          >
+                            <Link width="100%" href="/dashboard/chapters">
                               Dashboard
-                            </MenuItem>
-                          </NextLink>
+                            </Link>
```

Comment:
I have no idea why we used `NextLink`, I think it was because of error message in the concole but I can't find the PR, or replicate the issue. I should learn more about git